lib/aphorism.rb

```diff
@@ -4,9 +4,23 @@ module Aphorism
   class Error < StandardError; end
   class Orator
     def say
+      aphorisms = internal_aphorisms + custom_aphorisms
+      puts (aphorisms).sample
+    end
+
+    private
+    
+    def internal_aphorisms
       aphorisms_path = File.join(__dir__, 'content/aphorisms.txt')
-      aphorisms = IO.read(aphorisms_path).split("\n%\n")
-      puts aphorisms.sample
+      IO.read(aphorisms_path).split("\n%\n")
+    end
+
+    def custom_aphorisms
+      custom_path = File.join(Dir.home, '.aphorism')
+      custom_files = Dir.children(custom_path)
```

Comment:
If this directory doesn't exist, it will throw a Errno::ENOENT...

Reply:
Fixed in #8